numpy/core/_add_newdocs.py

```diff
@@ -4876,7 +4876,7 @@
 
 add_newdoc('numpy.core', 'ufunc', ('reduce',
     """
-    reduce(a, axis=0, dtype=None, out=None, keepdims=False, initial)
+    reduce(a, axis=0, dtype=None, out=None, keepdims=False, initial, *, where=True)
```

Comment:
This `*` is not true - I assume it's a remnant of the rejected suggestion to implement it this way.

Reply:
Oops, yes, corrected.